frontend/src/pages/search.tsx

```diff
@@ -1,26 +1,71 @@
 import type { GetStaticProps, NextPage } from "next";
-import { useFeatureFlags } from "src/hooks/useFeatureFlags";
+import {
+  Opportunity,
+  SearchFetcher,
+  fetchSearchOpportunities,
+} from "../services/searchfetcher/SearchFetcher";
+import React, { useState } from "react";
 
+import { APISearchFetcher } from "../services/searchfetcher/APISearchFetcher";
+import { MockSearchFetcher } from "../services/searchfetcher/MockSearchFetcher";
+import PageNotFound from "./404";
 import { serverSideTranslations } from "next-i18next/serverSideTranslations";
+import { useFeatureFlags } from "src/hooks/useFeatureFlags";
 
-import PageNotFound from "./404";
+const useMockData = true;
```

Comment:
We may want to leave a comment here with a generic TODO to remove this mockData logic once we are confident in live API data. Generally best practice in React apps is to move mockData to a separate component to be called during tests and storybook usage, but kept away from the production component for readabilities sake.